lib/ProductOpener/Food.pm

```diff
@@ -500,56 +500,56 @@ It is a list of nutrients names with eventual prefixes and suffixes:
 			'--arachidic-acid-', '--behenic-acid-',
 			'--lignoceric-acid-', '--cerotic-acid-',
 			'--montanic-acid-', '--melissic-acid-',
-			'-monounsaturated-fat-', '-polyunsaturated-fat-',
-			'-omega-3-fat-', '--alpha-linolenic-acid-',
-			'--eicosapentaenoic-acid-', '--docosahexaenoic-acid-',
-			'-omega-6-fat-', '--linoleic-acid-',
+			'-unsaturated-fat-', '--monounsaturated-fat-',
+			'---omega-9-fat-', '--polyunsaturated-fat-',
+			'---omega-3-fat-', '---omega-6-fat-',
```

Comment:
the fact to have four dashes does not harm however, it's displayed as 3 :-)

Reply:
![image](https://github.com/openfoodfacts/openfoodfacts-server/assets/144788/e7fbe562-5379-45f8-886e-070cc852ecee)